Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM google/debian:wheezy
 ENV DEBIAN_FRONTEND noninteractive
-RUN sed -i '1i deb     http://gce_debian_mirror.storage.googleapis.com/ wheezy         main' /etc/apt/sources.list
-RUN apt-get update && apt-get install -y -qq --no-install-recommends wget unzip python php5-mysql php5-cli php5-cgi openjdk-7-jre-headless openssh-client && apt-get clean
+RUN apt-get update && apt-get install -y -qq --no-install-recommends wget unzip python php5-mysql php5-cli php5-cgi openjdk-7-jre-headless openssh-client python-openssl && apt-get clean
```

Comment:
can you change apt-get clean,  with rm -rf /var/lib/apt/lists/*? I think google/debian already does apt-get clean after each install now.